packages/office-addin-dev-settings/src/cli.ts

```diff
@@ -84,7 +84,8 @@ commander
     .action(commands.webView)
     .on("--help", () => {
         console.log("\nFor [web-view-type], choose one of the following values:\n");
-        console.log("\t'edge' for Microsoft Edge");
+        console.log("\t'edgechromium' for the new Chromium-based Microsoft Edge");
```

Comment:
I think leave out "the new" because at some point it won't be new

Reply:
"newer"? or leave it out completely

Reply:
We should strive to be consistent with the naming. I think we should transition to calling the older version `legacy Microsoft Edge` which is how the https://www.microsoft.com/en-us/edge page puts it. For the new version, I think we should call it `Microsoft Edge (Chromium)`. It is also possible to refer to the legacy version as `Microsoft Edge (EdgeHTML)`.

Reply:
I would recommend to update this to:
```
`edge-legacy` for the legacy Microsoft Edge (EdgeHTML)`
`edge` or `edge-chromium` for Microsoft Edge (Chromium)`
```

I think we could update the code to follow these strings for the versions. Notably, we should switch `edge` to refer to chromium now (it's simpler), and modify `edgechromium` to `edge-chromium` keeping both for backward compatibility but documenting the one with the hyphen.

I'd like your views on this, but I think this would make more sense going forward.

Reply:
This should be an easy change, but my immediate concern is it might break some workflows for users. Do we do a minor version bump to account for this, then?

Reply:
It is fair to consider if this will impact users -- do we have any data about how much this command is used? Also, we should see if there is documentation about this and consider that we'll need to update docs to match. This is just for the `edge` tag as the others won't affect anything. We could leave `edge` for the legacy Edge but I really think it makes sense at some point to switch it. The semantic version of the package really won't make a different. We could make a major version bump as a breaking change I don't think this warrants it.  @TCourtneyOwen what's your view? Should we leave `edge` for Edge legacy?  We can still go ahead and add `edge-legacy` and `edge-chromium` and perhaps update docs to refer to them since it is more clear.

Reply:
At this point, would the PR be ready for approval? @TCourtneyOwen @akrantz

Reply:
I can review the PR with the assumption we're going to switch `edge` to Edge Chromium, but we'll need to make a call there and then make the minor changes needed if we don't want that. 

@arttarawork It would be great if you could:
1. get data about the usage of the webview command to know if significant.
2. point out the relevant documentation so we can determine if the docs would be affected by this change.

Reply:
Let me see if I can quickly get that data and I will send a query url with the results

Reply:
It appears that we don't currently collect any usage data for office-addin-dev-settings.  Nor do we collect and related data in offfice-addin-debugging.  We should add a work item to add usage data to office-addin-dev-settings

Reply:
As for existing documentation for switching the webview, I couldn't find anything when doing a search, so it's quite likely our ReadMe is our only documentation